### ISSUE TITLE
Create and start using a development branch

### ISSUE THREAD

**Issue by kristoferlund** [givepraise/praise#1102]
We should consider using a `development` branch, right? And configure things like this:

Dev flow:
- Code, make PR, merge with `development`
- Code, make PR, merge with `development`
- Code, make PR, merge with `development`
- Build production images (Github Action)
- Upgrade staging server
- Test run on staging server
- All good, make PR from `development` to `main``
- Mark version
- Build production images (Github Action)
- Upgrade production server

Changes needed:
- Create branch `development``
- Change base for exisitin WIP PRs, from `main` to `development`
- Development Docker images are built from branch `development` instead as now, from `main`. The GitHub Actions already exist: `package-dev-api.yml`etc

**Comment by kkatusic** [comment]
@kristoferlund I agree with this it would be more easy to test some stuff when there an new code is online and when you approve pull request that code can be pushed immediately to staging domain: https://staging.dev.givepraise.xyz/ and maybe open two client for testing...

How hard is it to do that work?
